assets/pages/digital-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -131,6 +135,25 @@ function CheckoutForm(props: PropTypes) {
           setValue={props.setLastName}
           error={firstError('lastName', props.formErrors)}
         />
+        <InputWithFooter
+          id="email"
+          label="Email"
+          type="email"
+          disabled
+          value={props.email}
+          footer={(
+            <span>
+              <CheckoutExpander copy="Want to use a different email address?">
+                <p>You will be able to edit this in your account once you have completed this checkout.</p>
+              </CheckoutExpander>
+              <CheckoutExpander copy="Not you?">
+                <p>
+                  <Button appearance="greyHollow" icon={null} type="button" aria-label={null} onClick={() => props.signOut()}>Sign out</Button> and create a new account.
```

Comment:
Could `icon={null}` be the default behaviour of `Button`?

Reply:
it's confusing to read, so many of our button instances just have the arrow in there that at the time it felt like a nice syntax shortcut. Now that contributions is using this button too it's a very good point to raise though!